aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -888,7 +902,8 @@ Tensor outer(const Tensor& self, const Tensor& vec2) {
 }
 
 static void addmm_impl_cpu_(
-    Tensor &result, const Tensor &self, Tensor m1, Tensor m2, const Scalar& beta, const Scalar& alpha) {
+    Tensor &result, const Tensor &org_self, Tensor m1, Tensor m2, const Scalar& beta, const Scalar& alpha) {
+  auto self = *(expand_size(org_self, {m1.sizes()[0], m2.sizes()[1]}, "addmm_out"));
```

Comment:
@swolchok is it time for you to roll in your... err... ergo chair?

Reply:
yeah I wasn't sure why I did this... let me change it.